alpha/apps/kaltura/lib/storage/urlTokenizers/kUrlTokenizer.php

```diff
@@ -77,14 +77,14 @@ public function tokenizeMultiUrls(&$baseUrl, &$flavors)
 	}
 	
 	/**
-	 * @return the $window
+	 * @return number $window
```

Comment:
@TalDubovKaltura you can probably set this type to int.

Reply:
Done